package.json

```diff
@@ -4,14 +4,16 @@
   "description": "Barefoot Nomad - Making company travel and accomodation easy and convenient. =======",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test npm run migrate:reset && cross-env NODE_ENV=test  npm run migrate && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout 10000 -c --require @babel/register --exit",
+    "test": "cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout 10000 -c --require @babel/register --exit",
```

Comment:
```suggestion
    "test": "pretest && cross-env NODE_ENV=test nyc --reporter=text --reporter=html mocha --timeout 10000 -c --require @babel/register --exit",
```

Reply:
@King-Success I suggest you add these changes
This would help automate the test

Reply:
@chis0m running `npm run test` command automatically triggers `npm run pretest` first before it continues. There's no need to it again

Reply:
ok